taxonomies/additives.txt

```diff
@@ -16422,63 +16509,77 @@ additives_classes:en: en:sequestrant
 vegan:en:yes
 vegetarian:en:yes
 
-en:E578, Calcium gluconate
+comment:en:CALCIUM GLUCONATE is for the moment listed in both the additves and ingredients taxonomy.
+#<en:Calcium
+<en:E578
+en:E578, calcium gluconate
```

Comment:
Guessing it doesn't like this.
2020-09-07T15:43:53.8871819Z Errors in the additives taxonomy definition:
2020-09-07T15:43:53.8909631Z ERROR - en:e578 is a parent of itself
2020-09-07T15:43:53.8910404Z Errors in the additives taxonomy definition at lib/ProductOpener/Tags.pm line 1618.
2020-09-07T15:43:54.1858128Z [additives] child process exited with code 25